yarn-project/simulator/src/acvm/oracle/oracle.ts

```diff
@@ -171,6 +172,27 @@ export class Oracle {
     return capsule.map(toACVMField);
   }
 
+  async getPublicKeysAndPartialAddress([address]: ACVMField[]): Promise<ACVMField[]> {
```

Comment:
The function name and the ordering of the return values messes with my brain. If we have the the key first in the name I would also assume it is also first in the return values.